fla/models/mom_gated_deltanet/configuration_mom_gated_deltanet.py

```
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Optional
+
+from transformers.configuration_utils import PretrainedConfig
+
+
+class MomGatedDeltaNetConfig(PretrainedConfig):
+    model_type = 'mom_gated_deltanet'
+    keys_to_ignore_at_inference = ['past_key_values']
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (32/5)

(R0913)